bindata/network/multus-admission-controller/admission-controller.yaml

```diff
@@ -0,0 +1,63 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: multus-admission-controller
+  namespace: openshift-multus
+  labels:
+    app: multus-admission-controller
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the Multus admisson controller component on each node.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  selector:
+    matchLabels:
+      app: multus-admission-controller
+      namespace: openshift-multus
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: multus-admission-controller
+        namespace: openshift-multus
+        component: network
+        type: infra
+        openshift.io/component: network
+    spec:
+      containers:
+      - name: multus-admission-controller
+        image: {{.MultusAdmissionControllerImage}}
+        command:
+        - /usr/bin/webhook
+        args:
+        - -bind-address=0.0.0.0
+        - -port=443
+        - -tls-private-key-file=/etc/webhook/tls.key
+        - -tls-cert-file=/etc/webhook/tls.crt
+        - -alsologtostderr=true
+        volumeMounts:
+        - name: webhook-certs
+          mountPath: /etc/webhook
+          readOnly: True
+        imagePullPolicy: IfNotPresent
+        resources:
+          requests:
+            cpu: 10m
+      serviceAccountName: multus
+      priorityClassName: "system-cluster-critical"
+      restartPolicy: Always
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      volumes:
+      - name: webhook-certs
+        secret:
+          secretName: multus-admission-controller-secret
+      tolerations:
+      - key: "node-role.kubernetes.io/master"
+        operator: Exists
+        effect: NoSchedule
+      - key: "node.kubernetes.io/not-ready"
```

Comment:
I don't think you can tolerate this, since your pod isn't HostNetwork.

Reply:
As the conclusion of OpenShift plumbing meeting, we decide to run admission controller for all master nodes (for HA). So DaemonSet + master toleration could be possible.

Could you please let me know how to run Pod for all master node (that will be configured so we cannot use '3' as fixed number) with ReplicaSet, if you know about it? I will follow you.